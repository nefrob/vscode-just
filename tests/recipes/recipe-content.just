# Noise operators

@quiet:
  echo hello

_private:
  echo hello

body:
  -echo "suppress error"
  @echo quiet
