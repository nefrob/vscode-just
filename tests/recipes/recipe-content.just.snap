># Noise operators
#^^^^^^^^^^^^^^^^^ source.just comment.line.number-sign.just
>
>@quiet:
#^ source.just keyword.other.recipe.prefix.just
# ^^^^^ source.just entity.name.function.just
#      ^ source.just keyword.operator.recipe.end.just
>  echo hello
#^^^^^^^^^^^^^ source.just
>
>_private:
#^ source.just keyword.other.recipe.prefix.just
# ^^^^^^^ source.just entity.name.function.just
#        ^ source.just keyword.operator.recipe.end.just
>  echo hello
#^^^^^^^^^^^^^ source.just
>
>body:
#^^^^ source.just entity.name.function.just
#    ^ source.just keyword.operator.recipe.end.just
>  -echo "suppress error"
#^^ source.just
#  ^ source.just keyword.operator.error-suppression.just
#   ^^^^^ source.just
#        ^ source.just string.quoted.double.just string.quoted.double.just
#         ^^^^^^^^^^^^^^ source.just string.quoted.double.just
#                       ^ source.just string.quoted.double.just
>  @echo quiet
#^^ source.just
#  ^ source.just keyword.operator.quiet.just
#   ^^^^^^^^^^^ source.just
>